tools/meson.build

```diff
@@ -18,3 +18,9 @@ if not nitrogfx_exe.found()
     subproject('nitrogfx', default_options: 'native=true')
     nitrogfx_exe = find_program('nitrogfx', native: true)
 endif
+
+constgen_py = find_program('constgen_py', native: true, required: false)
```

Comment:
Can't you use the `fallback:` parameter? Granted, that's a new meson feature I think.

Reply:
This isn't really a `fallback` candidate, since it's not a different name. We're expecting to find it under this name.